cloud_testing/xctest_parser/src/test/kotlin/xctest/ParseTest.kt

```diff
@@ -47,6 +47,21 @@ class ParseTest {
             "testWithGreyAssertions"
     )))
 
+    init {
+        val releaseUrl = "https://github.com/Flank/test_artifacts/releases/download/v0.2/EarlGreyExampleTests.zip"
+        val fixtures = File("./src/test/kotlin/xctest/fixtures")
+        if (!fixtures.exists()) {
```

Comment:
instead of checking for the zip, we should check md5 of the zip to determine changes. The same release can be updated multiple times.